.circleci/src/jobs/deploy-stage-nodes.yml

```diff
@@ -17,11 +18,11 @@ steps:
         git config --global user.email "audius-infra@audius.co"
         git config --global user.name "audius-infra"
   - run:
-      name: Update tag in audius-docker-compose
+      name: Update tag in audius-docker-compose stage branch
       command: |
         git clone --branch stage https://github.com/AudiusProject/audius-docker-compose.git audius-docker-compose
         cd audius-docker-compose
-        sed -i "s/{TAG:-.*}/{TAG:-$CIRCLE_SHA1}/" */docker-compose.*.yml
```

Comment:
this was a bug introduced [here](https://github.com/AudiusProject/audius-protocol/commit/b80fe9ab5f099668b7ab8fcddebd74ffeeb8140d). it was intended to include `discovery-provider/docker-compose.plugins.yml` but it also excluded the other docker-compose.yml files